plutus-core/plutus-core/src/PlutusCore/DeBruijn/Internal.hs

```diff
@@ -45,14 +47,12 @@ import qualified Data.Bimap               as BM
 import qualified Data.Text                as T
 import           Prettyprinter
 
-import           Numeric.Natural
-
 import           Control.DeepSeq          (NFData)
 import           ErrorCode
 import           GHC.Generics
 
 -- | A relative index used for de Bruijn identifiers.
-newtype Index = Index Natural
+newtype Index = Index Word
```

Comment:
This could also be made Int. I use Word because in BRAList we used Word as well. But currently in the `RelativizedMap` we use IntMap, so perhaps Int is a better fit afterall.

Reply:
i reverted Index back to Natural for now